lib/Cake/Controller/Component/PaginatorComponent.php

```diff
@@ -80,14 +80,30 @@ class PaginatorComponent extends Component {
  * - `paramType` What type of parameters you want pagination to use?
  *      - `named` Use named parameters / routed parameters.
  *      - `querystring` Use query string parameters.
+ * - `queryScope` By using request parameter scopes you can paginate multiple queries in the same controller action.
```

Comment:
@markstory PaginatorComponent::paginate() already has a parameter named **$scope,** it will be confusing and it created problems when trying: `$paginator->paginate = ['scope' => 'articles]`.

https://github.com/cakephp/cakephp/blob/2.next/lib/Cake/Controller/Component/PaginatorComponent.php#L128